apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-form.tsx

```diff
@@ -85,10 +85,10 @@ let triggerPreviousForm: () => void;
             validate={ (values): any => {
                 const errors:any = {};
                 if (!values.scopeName && !initialValues?.scopeName) {
-                    errors.scopeName="Required";
+                    errors.scopeName="This field cannot be empty";
```

Comment:
Shall we add `i18n` strings here.

**Ref** - https://github.com/wso2/identity-apps/blob/master/modules/i18n/src/translations/en-US/portals/console.ts#L2905